app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java

```diff
@@ -38,6 +38,6 @@
      */
     Task<Event> updateEvent(Event event);
 
-    Task deleteEvent(Event event);
+    Task<Event> deleteEvent(Event event);
```

Comment:
why tho, Task<Void> would be better I think

Reply:
Yes you're right

Reply:
Task<Void> I meant

Reply:
Ahh yes! It's better for sure! I'll fix this